software/index.html

```diff
@@ -36,96 +37,139 @@
-                            This project is great on a resume or portfolio and will spark interest from recruiters and hiring managers!
-                        </p>
-                    </div>
-                    <div class="col-md-2 align-self-center">
-                        <img class="img-fluid" src="images/react_logo.svg" alt="ReactJS Logo">
                     </div>
                 </div>
-                <p class="fs-5 text-center">Contact us at <a class="link-dark fw-bold" href="mailto:software-saturdays@purdueieee.org">software-saturdays@purdueieee.org</a> with questions.</p>
-                <div class="d-grid gap-2 d-sm-flex justify-content-sm-center text-center">
-                    <a href="/software/attendee/" type="button" class="btn btn-ieee btn-lg px-4 gap-3">Register Today!</a>
-                </div>
             </div>
 
-            <div class="my-2 p-5 container-fluid bg-purdue">
-                <div class="container-lg">
-                    <h3 class="text-center">Frequently Asked Questions:</h3>
-                    <ul class="fs-4">
-                        <li class="my-3"><span class="fw-bold">I am not in the College of Engineering, can I still join?</span><br>Of course, any Purdue student can join, regardless of major, college, or class standing!</li>
-                        <li class="my-3"><span class="fw-bold">How much time does this program require?</span><br>Aside from the two hour weekly learning session, most people spend only 1-2 hours reviewing material.</li>
-                        <li class="my-3"><span class="fw-bold">How much prior experience do I need?</span><br>Absolutely none. We teach you what you have to know! If you are familiar with web development, we do offer an intermediate track that may be more your style.</li>
-                        <li class="my-3"><span class="fw-bold">What is this certificate you mentioned?</span><br>Any attendee who completes our final project will recieve a digital certificate from the College of Engineering,
-                            with a QR code to verify your achievement.</li>
-                    </ul>
+            <!-- Our Sponsors -->
+            <div class="mt-2 mb-5 container-fluid text-light text-center fade-in-top">
+                <h1 class="text-center mb-3"><strong>Our Sponsors</strong></h3>
+                <div class="container bg-pieee extra-rounded d-flex flex-row justify-content-evenly align-items-center mb-5 py-5">
+                    <div class="col">
+                        <img class="w-50" src="./images/coe_logo.png" alt="College of Engineering Logo">
+                    </div>
+                    <div class="col">
+                        <img class="w-50" src="./images/IEEE_logo.png" alt="IEEE Logo">
```

Comment:
On mobile, these images do not scale properly. Try to have them stack vertically with a flex box once they are below a threshold. Look at https://dev.purdueieee.org/rov/sponsors/.